### DISCUSSION TITLE
[Feature] option to delete only the local copy of photo

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#4453] [Feature Request]
### The feature

When clicking the trashcan button 🗑️, it would be nice to have an option to delete only the local copy of photo, when the photo is uploaded on the server already.

Now it's a bit confusing as it deletes both by default.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by CurlyFlow** [comment]
Why should "delete" doenst mean delete? LOL Thats what delete does. It deletes it. Would be quite confusing if you delete something and its still there.

**Reply by jjspierx** [reply]
Deleting a local file is very different than deleting the remote file. In this instance, there needs to be a distinction. Many users will want to free up space on their device while keeping the images on the server.

**Reply by CurlyFlow** [reply]
DId u ever use google photos? If u delete something, it should get deleted, what you want is a free space option (again look at google)

**Comment by schmitzkr** [comment]
Is this the same as https://github.com/immich-app/immich/discussions/2379

**Reply by shenlong-tanwen** [reply]
Yes, Thank you for linking the relevant discussion. I'll mark this as a duplicate.

**Comment by shenlong-tanwen** [comment]
Duplicate of #2379
